docs/react-three-fiber/API/canvas.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Canvas
+description: 'The Canvas object is your portal into Threejs.'
+nav: 0
+---
+
+The `Canvas` object is where you start to define your React Three Fiber Scene.
+
+```jsx
+import React from 'react'
+import { Canvas } from '@react-three/fiber'
+
+const App = () => (
+  <Canvas>
+    <pointLight position={[10, 10, 10]} />
+    <mesh>
+      <sphereBufferGeometry />
+      <meshStandardMaterial color="hotpink" />
+    </mesh>
+  </Canvas>
+)
+```
+
+The props available for the canvas are:
+
+| Prop         | Type                                  | Default                                                  |
+| ------------ | ------------------------------------- | -------------------------------------------------------- |
+| children     | ThreeeJS elements or React Components |                                                          |
+| gl           | Object                                | `{}`                                                     |
```

Comment:
Maybe worth adding that this object is directly related to the args of `THREE.WebGLRenderer`? IIRC this can also be a Renderer e.g. SVG

Reply:
agreed, but lets fix it in another pass. i hope i have time tonight, then i can do it